src/index.test.js

```diff
@@ -72,6 +72,19 @@ test('shows dropdown', t => {
   t.snapshot(toJson(wrapper))
 })
 
+test('always shows dropdown', t => {
+  const { tree } = t.context
+  const wrapper = shallow(<DropdownTreeSelect id={dropdownId} data={tree} showDropdownAlways />)
+  t.snapshot(toJson(wrapper))
+})
+
+test('keeps dropdown open  for showDropdownAlways', t => {
```

Comment:
The extra space in the title can be removed.

Reply:
Other than that, this looks OK. @ellinge thoughts?

Reply:
@mrchief  hi, which line exactly are you referring to when you said "The extra space in the title"?

Reply:
yap. deleted the extra space.

Reply:
Probably could avoid some event handling etc if's always open, but I guess this is simple enough so never mind.

We have to make sure it's not closed with keyboard in some way also in the other PR, I think that it won't be a problem though. Not sure if we need to make any changes with ARIA as well but that's not part of this at this moment at least.